Doc/glossary.rst

```diff
@@ -221,6 +221,14 @@ Glossary
       statement by defining :meth:`__enter__` and :meth:`__exit__` methods.
       See :pep:`343`.
 
+    context variable
+      A variable which can have different values depending on it's context.
+      This is similar to TLS(Thread Local Storage) in which each execution
+      thread may have a different value for a variable. In case of context
```

Comment:
s/In case of context/In the case of a context

Reply:
i have re-written that part of sentence which Iam now satisfied with.
Kindly let me know ur comments.